skills/tool-design/scripts/description_generator.py

```
@@ -28,210 +96,433 @@
+
+        "RATE_LIMITED": json.dumps({
             "error": "{error_code}",
             "message": "Rate limit exceeded",
-            "retry_after": {seconds},
-            "resolution": "Wait {seconds} seconds before retrying"
-        }}
-        """
+            "retry_after": "{seconds}",
+            "resolution": "Wait {seconds} seconds before retrying",
+        }, indent=2),
     }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)